code/modules/mob/living/silicon/robot/sprites/_sprite_datum.dm

```diff
@@ -88,7 +112,7 @@
 	has_dead_sprite_overlay = TRUE
 	has_custom_equipment_sprites = TRUE
 	pixel_x = -16
-
+/* //Does not need to be dogborg-only, letting all borgs use these -Reo
```

Comment:
All borgs don't actually HAVE resting sprite styles for alternative options, is why this specific selection is in there, they don't even have default sprite-rest sprites.

Reply:
On this... They might not have resting sprites, but if the intent was to merg dogborgs with normal borgs, I think requiring all forward going borgs to have to be dogborg subtypes to use resting sprites is kinda ick and against the point of merging the two, thus this will allow borgs going forward to use what dogborgs have and not have to be dogborg subtypes. Arguably this could also be used to make dogborgs not a subtype outside of their specialized, dog-like differences (like having jaws for crowbars)

Reply:
Basically, dogborg subtype was made entirely for purpose of "this is set of features that old dogborg modules had and is common among all those sprites". New sprites can have that individually. Regardless, my concern for this portion is that it will break sprites for regular borgs if they rest for whatever reason, hence why I say this should stay as is. Plus you can see with tallborgs and such that you can just make 'individualized', per-sprite-type procs for selecting the resting sprite so it wont be necessary to be subtype of dogborg to do that.

Reply:
the rest code checks for if they have resting sprites, which by default is False and skips that check. Unless I missed something, but I'm pretty sure I had this issue and fixed it before. I'll test later